include/odp/api/spec/proto_stats.h

```diff
@@ -0,0 +1,166 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ * Copyright(C) 2021 Marvell.
+ */
+
+/**
+ * @file
+ *
+ * ODP Proto Stats
+ */
+
+#ifndef ODP_API_SPEC_PROTO_STATS_H_
+#define ODP_API_SPEC_PROTO_STATS_H_
+#include <odp/visibility_begin.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/** @addtogroup odp_proto_stats
+ *  @{
+ */
+
+/**
+ * @def ODP_PROTO_STATS_INVALID
+ * Invalid proto stats handle
+ */
+
+/** ODP proto stats id */
+typedef enum odp_proto_stats_id_t {
+	/** Max */
+	ODP_PROTO_STATS_ID_MAX,
+} odp_proto_stats_id_t;
+
+/** ODP proto stats params */
+typedef struct odp_proto_stats_param_t {
+	/** Stats id mask
+	 *
+	 * Each bit corresponds to one ODP_PROTO_STATS_ID_* and
+	 * bit position is same as ODP_PROTO_STATS_ID_*.
+	 * Bits set in this mask are the stat counters to be enabled
+	 * in the created proto stats object.
+	 */
+	uint64_t stats_mask;
+} odp_proto_stats_param_t;
+
+/**
+ * Proto stats capabilities
+ */
+typedef struct odp_proto_stats_capability_t {
+	/** Supported proto stats id mask
+	 *
+	 * A bit is set at position ODP_PROTO_STATS_ID_*,
+	 * when that proto stats id is supported.
+	 */
+	uint64_t stats_mask_supported;
+} odp_proto_stats_capability_t;
+
+/**
+ * Initialize proto stats parameters
+ *
+ * Initialize an odp_proto_stats_param_t to its default values.
+ * By default all the statistics are disabled.
+ *
+ * @param param   Proto stats parameter pointer.
+ */
+void odp_proto_stats_param_init(odp_proto_stats_param_t *param);
+
+/**
+ * Get proto stats capability
+ *
+ * Get proto stats capability of supported modes.
```

Comment:
The "of supported modes" part seems to be a leftover from earlier versions. It does not mean anything anymore and could be removed.